.travis/rpmbuild_test

```diff
@@ -8,21 +8,30 @@ yum install -y openssl-devel libuuid-devel
 
 # switch to an unprivileged user with sudo privileges
 yum install -y sudo
+# We need Git existing here prior to the run
+yum install -y git
 useradd -u 1000 --create-home -s /bin/bash testuser
 echo "Defaults:testuser env_keep=DOCKER_HOST" >>/etc/sudoers
 echo "testuser ALL=(ALL) NOPASSWD: ALL" >>/etc/sudoers
 chown -R testuser .
 
 su testuser -c '
   set -x
-  ./mconfig -V 0.0
+  set -e
+  ./mconfig 
   sudo yum-builddep -y dist/rpm/singularity.spec
+  git fetch --all
```

Comment:
I don't believe this fetch does anything, better to leave it out.

Reply:
:+1: